clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -1,5 +1,6 @@
 {{$PROMETHEUS_SCRAPE_KUBELETS := DefaultParam .PROMETHEUS_SCRAPE_KUBELETS false}}
 {{$PROMETHEUS_SCRAPE_WINDOWS_NODES := DefaultParam .PROMETHEUS_SCRAPE_WINDOWS_NODES false}}
+{{$PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER := DefaultParam .PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER false}}
```

Comment:
can we resuse the `PROMETHEUS_SCRAPE_WINDOWS_NODES`? feels awkward to have to different windows scrape settings

Reply:
PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER  is the variable we introduced when we switched to use Windows Exporter as a HostProcess and we need to use it for Azure jobs

PROMETHEUS_SCRAPE_WINDOWS_NODES  variable is used by the GCE jobs to install WMI exporter on the nodes via gcloud ssh command

I used the new PROMETHEUS_SCRAPE_WINDOWS_NODE_EXPORTER variable to add taints to the control-plane nodes and to remove storage configuration for Prometheus so I don't alter the other jobs that might be running Windows tests in GCE

Reply:
thanks for the clarification.  If we can eventually remove the GCE specific code and use just the hostprocess install method, we can eventually drop PROMETHEUS_SCRAPE_WINDOWS_NODES.  My understanding is that GCE doesn't support hpc currently but will in the future.